### ISSUE TITLE
Zigbee network visualization hard to read

### ISSUE THREAD

**Issue by xxv** [home-assistant/frontend#23968]
### Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

In the ZHA Zigbee network visualization:
![Image](https://github.com/user-attachments/assets/fdee272f-7de5-44ca-a929-ce564a0a335a)

The green and red colors are much too close in luminosity to be able to tell the nodes apart for someone who is red/green colorblind. As you can see in the image, the only data made available at that zoom level are the red/green shadings, so for someone who is red/green colorblind, it's hard to read the graph. I am mildly red/green colorblind and find it frustrating, but not impossible to read.

### Describe the behavior you expected

I recommend changing the green nodes to some other color, or using some other non-color way to signify that a node is offline (putting a little "!" near it? on it?). An alternative would be to make it so that the relative brightness/luminosity of the online vs. offline state are different. This is another clue that can help colorblind people distinguish between two states. Whatever solution is found can be tested using [the Chrome color vision deficiency dev tool](https://developer.chrome.com/docs/chromium/cvd) or others.

### Steps to reproduce the issue

1. Be red/green colorblind
2. Attempt to read the high-level overview of the ZHA graph, but find it lacking critical online/offline information for you
3. 
...

### What version of Home Assistant Core has the issue?

2025.1.4

### State of relevant entities

```txt

```

### Problem-relevant frontend configuration

```yaml

```

### Javascript errors shown in your browser console/inspector

```txt

```
